lib/assets/constants.dart

```diff
@@ -34,6 +34,7 @@ final TERMS_AND_CONDITIONS_URI = Uri.parse(
 
 final ABOUT_US_URI = Uri.parse('https://www.now-u.com/about');
 final FEEDBACK_FORM_URI = Uri.parse('https://forms.gle/xenzFkRVrELsqnnB9');
+final JOIN_RESEARCH_FORM_URI = Uri.parse('https://forms.gle/DjSdjH1DfNtRwXiP7');
```

Comment:
Just an idea for the future, maybe we can instead use links in `now-u` domain, like `https://www.now-u.com/links/join-research` and add redirection there to google forms. This will will help us to stay independend of the links generated by Google Forms and other services. Right now if the link would change we would need to push new app version and it won't work on older versions. Maybe there's a better way to handle it, this is what comes to my mind

Reply:
Yeah thats a great shout! Ive created a ticket to embed these forms in the website and will update these links once thats done.

Reply:
https://github.com/now-u/now-u-web/issues/87#issue-2650227262